### DISCUSSION TITLE
Looks like something went wrong! 500

### DISCUSSION THREAD

**Discussion by Munarbek-git** [community/community#76872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I go to the file view page, I get an error. Moreover, the file is from different repositories, as well as from your own repositories.
![image](https://github.com/community/community/assets/48122240/db9fb390-d181-403a-87b6-20885eaa7734)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Munarbek-git! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, are you still running into this issue on your end? Sometimes [clearing your cache and cookies](https://kb.iu.edu/d/ahic) does the trick. This also could have been caused by a service outage. You can check status.github.com to confirm in the future if a page load error you're seeing was caused by an outage.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by oap** [comment]
I am here to confirm this error and it happen to me almost all the time. 
When,
Anytime, when I logged in to view any repo. 
For example, https://github.com/microsoft is good.
https://github.com/microsoft/generative-ai-for-beginners will give a 500 error.

Clearing the cookies will do the trick if the repo is public but I loss access to any private repos requires login. 
For this issue, it might relate to specific accounts, since not many reports there. 

Anything else should I report? 
Browser: Chrome 120.0

**Reply by oap** [reply]
@Munarbek-git After contacting Support, I was directed to turn off some of my browser extensions and that resolved my issue. 
Hope this helps.

**Reply by Munarbek-git** [reply]
Thank you. Removing browser extensions helped in most cases.
